src/metadataGeneration/initializer-value.ts

```diff
@@ -1,7 +1,7 @@
 import * as ts from 'typescript';
 import { Tsoa } from './tsoa';
 
-export const getInitializerValue = (initializer?: ts.Expression, type?: Tsoa.Type) => {
```

Comment:
I would strongly prefer to keep the name as it is and only changing the underlying type declaration.

Reply:
Oh yea, I changed the name in the first place so that I could see (via compiler error messages) where I needed to update the code. Essentially I was using the compiler as my pair-programming buddy.

I fully intended to change the name back.

But honestly, I have no intention of merging a PR that makes development on the project harder. After all, refactor is supposed to:

* improve speed of development
  * it didn’t in this case because the new strictness caused some weirdness
* improve code quality
  * well, two tests are broken that I can’t fix so I can’t say that quality was improved

And most importantly, there’s the subjective quality of the work. If you would rather not work on top of this PR, then we shouldn’t merge it.

Either way, the name `Type` stays the same.